vm/actor_interface/src/builtin/account/mod.rs

```diff
@@ -42,6 +43,16 @@ pub fn is_v9_account_cid(cid: &Cid) -> bool {
     known_cids.contains(cid)
 }
 
+pub fn is_v10_account_cid(cid: &Cid) -> bool {
+    let known_cids = vec![
+        // calibnet v10
+        Cid::try_from("bafk2bzaceavfgpiw6whqigmskk74z4blm22nwjfnzxb4unlqz2e4wg3c5ujpw").unwrap(),
```

Comment:
The v10 CIDs are not the same as the V9 CIDs. You can find the new CIDs here: https://github.com/filecoin-project/lotus/blob/392d3d4b8ae2c1177593da205f77464c74c2e728/build/builtin_actors_gen.go#L112

Reply:
Updated